src/helpers/LatexHelpers.ts

```diff
@@ -0,0 +1,67 @@
+import { CacheType, CommandInteraction, MessageAttachment } from "discord.js";
+import svg2img, { ResvgRenderOptions } from "svg2img";
+var mjAPI = require("mathjax-node");
+
+
+export const Santinize = (message: string): string => {
+    let res_commands = [
+        [`\text{`, `\\backslash text~{`],
+        ['$', `\\$`],
+        ['"', '\\"'],
+        ['\\union','\\cup']
+    ]
+
+    let cleanedMessage = message
+    res_commands.forEach(([searchValue, replaceValue]) => {
+        cleanedMessage = cleanedMessage.replace(searchValue, replaceValue)
+    })
+    return cleanedMessage
+
+}
+
+export const EquationRender = async (cleanedMessage: string, interaction: CommandInteraction<CacheType>) => {
+    mjAPI.config({
+        MathJax: {
+            SVG: {
+                // scale: 70,
+                //font: "STIX-Web",
+            },
+        }
+    });
+    mjAPI.start();
+
+    mjAPI.typeset({
+        math: cleanedMessage,
+        format: "inline-TeX", // or "Tex", "MathML"
```

Comment:
You can try `format: "Tex`, but I prefer this since I don't know much about Latex and this format seems prettier to me.

Reply:
So I looked up the difference. It is essentially the difference between `display: flex` and `display: inline-flex` in HTML. It doesnt matter at all with our usecase so whatever looks best IMO.